src/main/scala/loamstream/drm/Drmaa1Client.scala

```diff
@@ -87,8 +88,10 @@ final class Drmaa1Client(nativeSpecBuilder: NativeSpecBuilder) extends DrmaaClie
   override def killAllJobs(): Unit = {
     debug(s"Killing all jobs...")
 
-    killJob(Session.JOB_IDS_SESSION_ALL)
+    doKillJob(Session.JOB_IDS_SESSION_ALL)
   }
+  
+  private def doKillJob(id: String): Unit = withSession(_.control(id, Session.TERMINATE))
```

Comment:
Another "is this right?" given function name and parameter. Should this be named `killSession` instead?

Reply:
No, not really.  `Session`s are oddball DRMAA-specific implementation details.  I'm going to fold this method into `killAllJobs()`, now that `killJob()` is gone.